tests/test_mechanics.py

```diff
@@ -279,6 +279,31 @@ def test_discard_enchanted_cards():
 	assert not game.player1.hand
 
 
+def test_discover():
```

Comment:
Nope. Only heuristically. When I put those tests in the original discover PR jleclanche asked me to remove them, rightly I think, because you can only test it by iterating many times and checking that it has the expected type of cards.